readthedocs/core/logs.py

```diff
@@ -69,8 +68,14 @@ def __call__(self, logger, method_name, event_dict):
             "line.number": record.lineno,
         }
 
+        # NOTE: For some reason structlog isn't including
+        # the exception in the record.exc_info attribute,
+        # so we need to get it from the event_dict.
+        exc_info = event_dict.pop("exc_info", None)
```

Comment:
Why is it getting "popped" in stead of getting "getted"? :) If the existing return value (`return event_dict`) was expecting `exc_info`, then it's removed from the dict here..

Reply:
So, basically, we are replacing that value with the expected representation of the exception for new relic to render.

Reply:
Nice, thanks for the additional info!